crates/topos-sequencer-subnet-client/src/lib.rs

```diff
@@ -258,8 +257,11 @@ impl SubnetClient {
                 .parse()
                 .map_err(Error::WalletError)?
         } else {
-            // Dummy key, can not sign transactions
-            hex::encode([0u8; 32]).parse().map_err(Error::WalletError)?
+            // Dummy random key, can not sign transactions
+            // Could not be zero key
+            "bbb92d6e98884f76de468fa3f6278f8807c48bebc13595d45af5bdc4da7021aa"
+                .parse()
+                .map_err(Error::WalletError)?
```

Comment:
Seems overkill, but how about we do it randomly?

```rust    
let mut rng = rand::thread_rng();
let mut bytes = [0u8; 32];
rng.fill(&mut bytes[..]);

hex::encode(bytes);
```

Or we have a constant? I also worry about hardcoded "magic strings", even if they are obvious and commented! 

```rust
const DUMMY_MAGIC_KEY: &'static str = "bbb92d6e98884f76de468fa3f6278f8807c48bebc13595d45af5bdc4da7021aa"
```